### DISCUSSION TITLE
Has anyone worked on fine-tuning models for different English accents?

### DISCUSSION THREAD

**Discussion by NehaB18** [SWivid/F5-TTS#425] [General]
Thanks for the awesome model. 

Default accent looks like American English. Has anyone worked on fine-tuning models for different English accents? like Indian English or Irish English?

I am interested to know how much dataset and fine-tuning will be required to achieve this?

**Comment by lfhuhu** [comment]
Hello, I wonder if you have any experience in doing English with accent these days. Thank you very much

**Reply by piotrwitkowski** [reply]
Should be possible to fine-tune it with a good dataset.
I have some success, but probably would need more samples to make it work better. 
[FF5-TTS dwarwish/scottish](https://youtube.com/clip/UgkxxWFRStEXXajFzuhHyMxD6MzR7vva4MvP?si=xk_ILvkl87oYiqUO)

**Comment by piotrwitkowski** [comment]
As for the dataset size, I noticed that anything below 500 samples had issues with inference. That said, my fine-tuning with 1k samples performed better than the 8k one—I’d say it largely depends on dataset quality. Or maybe it’s just a skill issue on my end.
